packages/runtime/src/next/components/MakeswiftSlot.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
```

Comment:
We should define the context in a separate component that we can have a `<Slot>` RSC that takes the ID and calls `getComponentSnapshot` itself. The non-RSC version would be used only in Pages Router

Reply:
We're not doing this right now.